tfe/provider_test.go

```diff
@@ -286,23 +286,35 @@ func testAccPreCheck(t *testing.T) {
 }
 
 func testAccGithubPreCheck(t *testing.T) {
-	if GITHUB_TOKEN == "" {
+	if githubToken == "" {
 		t.Skip("Please set GITHUB_TOKEN to run this test")
 	}
-	if GITHUB_WORKSPACE_IDENTIFIER == "" {
+	if githubWorkspaceIdentifier == "" {
 		t.Skip("Please set GITHUB_WORKSPACE_IDENTIFIER to run this test")
 	}
-	if GITHUB_WORKSPACE_BRANCH == "" {
+	if githubWorkspaceBranch == "" {
 		t.Skip("Please set GITHUB_WORKSPACE_BRANCH to run this test")
 	}
 }
 
-var GITHUB_POLICY_SET_IDENTIFIER = os.Getenv("GITHUB_POLICY_SET_IDENTIFIER")
-var GITHUB_POLICY_SET_BRANCH = os.Getenv("GITHUB_POLICY_SET_BRANCH")
-var GITHUB_POLICY_SET_PATH = os.Getenv("GITHUB_POLICY_SET_PATH")
-var GITHUB_REGISTRY_MODULE_IDENTIFIER = os.Getenv("GITHUB_REGISTRY_MODULE_IDENTIFIER")
-var GITHUB_TOKEN = os.Getenv("GITHUB_TOKEN")
-var GITHUB_WORKSPACE_IDENTIFIER = os.Getenv("GITHUB_WORKSPACE_IDENTIFIER")
-var GITHUB_WORKSPACE_BRANCH = os.Getenv("GITHUB_WORKSPACE_BRANCH")
-var TFE_USER1 = os.Getenv("TFE_USER1")
-var TFE_USER2 = os.Getenv("TFE_USER2")
+func init() {
+	githubToken = os.Getenv("GITHUB_TOKEN")
+	githubPolicySetIdentifier = os.Getenv("GITHUB_POLICY_SET_IDENTIFIER")
+	githubPolicySetBranch = os.Getenv("GITHUB_POLICY_SET_BRANCH")
+	githubPolicySetPath = os.Getenv("GITHUB_POLICY_SET_PATH")
+	githubRegistryModuleIdentifer = os.Getenv("GITHUB_REGISTRY_MODULE_IDENTIFIER")
+	githubWorkspaceIdentifier = os.Getenv("GITHUB_WORKSPACE_IDENTIFIER")
+	githubWorkspaceBranch = os.Getenv("GITHUB_WORKSPACE_BRANCH")
+	tfeUser1 = os.Getenv("TFE_USER1")
+	tfeUser2 = os.Getenv("TFE_USER2")
```

Comment:
💅 Not a blocking problem, but some of these strings are common enough to be shadowed by locally scoped variables in the future. I think it would be both more safe and clear to rename all these envGithubToken, envTFEUser1, etc

Reply:
Good idea 💡